net/netclient/files/netclient.init

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh /etc/rc.common
+# Copyright 2024 Stan Grishin (stangri@melmac.ca)
+# shellcheck disable=SC3043,SC3060
+
+# shellcheck disable=SC2034
+START=95
+STOP=95
+# shellcheck disable=SC2034
+USE_PROCD=1
+
+readonly PKG_VERSION='dev-test'
+readonly packageName='netclient'
+readonly PROG=/usr/sbin/netclient
```

Comment:
I wonder why? We dont need it, because the command version is already provided by the binary.
Version dev-test is wrong and why you want to replace it each time with the sed in install section? :thinking:

Reply:
We do need it to match the version of the package which is not the same as version of the binary. The binary needs to present its version in a special way to be allowed to connect to netmaker.

Reply:
> We do need it to match the version of the package which is not the same as version of the binary. 

Then report it to upstream to make it more comfortable. Why we should tweak it here? 
How other GNU/Linux distros are doing that?

Reply:
To make the OpenWrt version of the package consistent with the OpenWrt style of versioning.